src/common/request.factory.js

```diff
@@ -129,7 +129,7 @@ export default /* @ngInject */ function (
       {
         field,
         comparator,
-        reference: angular.isArray(reference) ? reference.join(',') : reference,
+        reference,
```

Comment:
joining already done here for apiv7 : https://github.com/ovh-ux/ng-ovh-api-wrappers/blob/master/src/v7/request-upgrader.service.js#L39